### PR TITLE
more Jamstack 2021 queries

### PR DESCRIPTION
**Pull request by denar90**
progress on #2156

### CODE REVIEW

**Review by rviscomi — APPROVED** [review]

File: sql/2021/jamstack/framework_libraries.sql [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+WITH totals AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    COUNT(0) AS total
+  FROM
+    `httparchive.summary_pages.2021_07_01_*`
+  GROUP BY
+    _TABLE_SUFFIX
+),
+ssg AS (
```
Comment (rviscomi) [inline sql/2021/jamstack/framework_libraries.sql:13]:
```suggestion

ssg AS (
```

File: sql/2021/jamstack/framework_libraries.sql [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    url,
+    app as ssg_app
+  FROM
+    `httparchive.technologies.2021_07_01_*`
+  WHERE
+    LOWER(category) = "static site generator" OR
+    app = "Next.js" OR
+    app = "Nuxt.js"
+),
+js AS (
```
Comment (rviscomi) [inline sql/2021/jamstack/framework_libraries.sql:25]:
```suggestion

js AS (
```

File: sql/2021/jamstack/framework_libraries.sql [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    `httparchive.summary_pages.2021_07_01_*`
+  GROUP BY
+    _TABLE_SUFFIX
+),
+
+ssg AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    url,
+    app as ssg_app
```
Comment (rviscomi) [inline sql/2021/jamstack/framework_libraries.sql:18]:
```suggestion
    app AS ssg_app
```

File: sql/2021/jamstack/framework_libraries.sql [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    LOWER(category) = "static site generator" OR
+    app = "Next.js" OR
+    app = "Nuxt.js"
+),
+
+js AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    url,
+    app as js_app
```
Comment (rviscomi) [inline sql/2021/jamstack/framework_libraries.sql:31]:
```suggestion
    app AS js_app
```

**Comment by denar90** [comment]
@rviscomi thanks for fixing listings 💪

**Pull request merged**